handlers/graph.go

```diff
@@ -181,20 +184,50 @@ func buildNamespaceTrafficMap(namespace string, o options.Options, client *prome
 	// fetch the internally originating request traffic time-series
 	intVector := promQuery(query, time.Unix(o.QueryTime, 0), client.API())
 
+	// 4) query for TCP  traffic. Analogously to HTTP, three queries:
```

Comment:
I'd still like these moved under the HTTP handling so that those queries number 1 through 3, then the optional istio 4 and 5 queries.  After that the TCP section renumbered 1-3.

Reply:
Done!